travvy-server/routes/routes.js

```diff
@@ -1,6 +1,5 @@
 const RecommendController = require('../src/controllers/RecommendController.js')
 const CitySelectController = require('../src/controllers/CitySelectController.js')
-const UsersController = require('../src/controllers/UsersController.js')
```

Comment:
Looks great thanks for fixing this

Reply:
This should solve the error when starting the server